packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show lerpDouble;
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+
+import 'theme.dart';
+
+/// Defines the properties of [Tooltip] widgets.
+///
+/// Used by [TooltipTheme] to control the properties of tooltips in a widget
+/// subtree.
+///
+/// To obtain the current [TooltipTheme], use [TooltipTheme.of].
+///
+/// See also:
+///
+///  * [TooltipTheme], which describes the actual configuration of a
+///    tooltip theme.
+class TooltipThemeData extends Diagnosticable {
+  /// Creates the set of properties used to configure [Tooltip]s.
+  const TooltipThemeData({
+    this.height,
+    this.padding,
+    this.verticalOffset,
+    this.preferBelow,
+    this.excludeFromSemantics,
+    this.decoration,
+    this.textStyle,
+    this.waitDuration,
+    this.showDuration,
+  });
+
+  /// They height of the tooltip's [child].
+  ///
+  /// If the [child] is null, then this is the intrinsic height.
+  final double height;
+
+  /// The amount of space by which to inset the child.
+  ///
+  /// Defaults to 16.0 logical pixels in each direction.
+  final EdgeInsetsGeometry padding;
+
+  /// The vertical gap between the widget and the displayed tooltip.
+  final double verticalOffset;
+
+  /// Whether the tooltip defaults to being displayed below the widget.
```

Comment:
defaults to being displayed below the widget => is displayed below its widget by default